indexer-app/src/cmr/indexer/services/index_service.clj

```diff
@@ -268,8 +284,10 @@
                                        nil)))
                              (remove nil?))
         humanized-fields (map #(humanizer/collection-humanizers-elastic context %) parsed-concepts)
-        suggestion-docs (map get-suggestion-docs humanized-fields)]
-    (flatten suggestion-docs)))
+        suggestion-docs (->> (map get-suggestion-docs humanized-fields)
+                             flatten
```

Comment:
Its hard to tell, but do we need flatten here, or can mapcat be used instead?

Reply:
If the result is unaffected I can simplify

Reply:
I think I tried to use mapcat a while ago but couldn't get it to work correctly so I went with flatten